coffee/remote_peer.coffee

```diff
@@ -52,11 +53,11 @@ class palava.RemotePeer extends palava.Peer
     options = []
     if @room.options.stun
       options.push({urls: [@room.options.stun]})
-    if @room.options.turn
+    if @room.options.turn && @turnCredentials
       options.push
```

Comment:
Only being able to add one TURN configuration is limiting. A common scenario is adding one TURN server using UDP and one server using TCP (with and/or without TLS). Physically this can be the same TURN server, just a different configuration.

Reply:
Thanks, I changed that in the latest commit